### PR TITLE
Add missing RBAC permission to kube-state-metrics

### PR DESCRIPTION
**Pull request by linki**
It now looks for VPAs and needs permission.

### CODE REVIEW

File: cluster/manifests/kube-state-metrics/rbac.yaml [hunk 1]
```diff
@@ -78,6 +78,7 @@ rules:
   - autoscaling
   resources:
   - horizontalpodautoscalers
+  - verticalpodautoscalers
```
Comment (mikkeloscar) [inline cluster/manifests/kube-state-metrics/rbac.yaml]: Does this work? I would imagine `autoscaling.k8s.io` is a different `apiGroup` than `autoscaling`?

Comment (linki) [reply cluster/manifests/kube-state-metrics/rbac.yaml]: That's what it claimed it needs in the error message.

Comment (linki) [reply cluster/manifests/kube-state-metrics/rbac.yaml]: Ah, no, `autoscaling.k8s.io` is not equal to `autoscaling` I guess :face_with_head_bandage:

Comment (linki) [reply cluster/manifests/kube-state-metrics/rbac.yaml]: Hopefully fixed.

**Comment by linki** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**